homeworks/valitothattila/2_vars_datatypes/fictive_character.py

```diff
@@ -0,0 +1,23 @@
+"""
+Feladat 1: Változók, user input, string metódusok, type conversion, f-string használata
+"""
+#adatok bekérése
+in_name = input("Enter your name: ")
+in_age = input("How old are you: ")
+in_python_exp_in_years = input("Years of Python experience: ")
+
+#vizsgálatok
+name = str.strip(in_name) #szóköz levágása elöl/hátul
```

Comment:
Szia!
érdemes a .strip() és capitalize() metódusokat direktben a változóra hívni, nem a string osztályra, és átadni neki a változót, egyszerűbb:
`str.strip(in_name) `helyett csak simán` in_name.strip()`
ugyanez igaz a capitalize()-ra is.

Még egyszerűbb,ha mindent megcsinálsz már az input-nál az 5. soron:

`in_name = input("Enter your name: ").strip().capitalize() `